rust/agents/scraper/migration/src/m20220805_000003_create_table_transaction.rs

```diff
@@ -0,0 +1,92 @@
+use sea_orm_migration::prelude::*;
+
+use crate::l20220805_types::*;
+use crate::m20220805_000002_create_table_block::Block;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Transaction::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Transaction::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
```

Comment:
Is Hash + chain useful? It is already unique by hash and it was my understanding that transactions include chain ids to prevent duplicate hashes across chains.

Also, as before, still can't use that as the ID.

Reply:
Good point and okay noted